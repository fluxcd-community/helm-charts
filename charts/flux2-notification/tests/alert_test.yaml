suite: test alert deployment
templates:
  - alert.yaml
tests:
  - it: should be empty by default
    asserts:
      - hasDocuments:
          count: 0
  - it: should match snapshot
    asserts:
      - matchSnapshot: {}
    set:
      alertlist:
      - name: all-kustomizations
        spec:
          providerRef:
            name: dev-msteams
          eventSeverity: error
          eventSources:
            - kind: Kustomization
              namespace: default
              name: '*'
          suspend: true
