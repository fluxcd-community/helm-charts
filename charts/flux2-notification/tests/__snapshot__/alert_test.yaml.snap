should match snapshot:
  1: |
    apiVersion: notification.toolkit.fluxcd.io/v1beta1
    kind: Alert
    metadata:
      labels:
        app.kubernetes.io/instance: NAMESPACE
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/part-of: flux
        app.kubernetes.io/version: 0.31.3
        helm.sh/chart: flux2-notification-1.0.0
      name: all-kustomizations
      namespace: NAMESPACE
    spec:
      eventSeverity: error
      eventSources:
      - kind: Kustomization
        name: '*'
        namespace: default
      providerRef:
        name: dev-msteams
      suspend: true
